### PR TITLE
Configurable Event Proxy Changes for Migration

### PR DESCRIPTION
**Pull request by amitkumar94**

### CODE REVIEW

File: client/pom.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@
     <parent>
         <groupId>com.flipkart</groupId>
         <artifactId>flux</artifactId>
-        <version>1.1.4-SNAPSHOT</version>
+        <version>1.2.0-SNAPSHOT</version>
```
Comment (shyamakirala) [inline client/pom.xml]: Not part of this one, but delete the earlier 1.2.0-SNAPSHOT git tag, create again as the old code results in compilation errors

File: runtime/src/main/resources/packaged/configuration.yml [hunk 2]
```diff
@@ -79,5 +80,9 @@ eventScheduler:
     batchSize: 50
     intervalms: 10000
 
+eventProxyForMigration:
+  enabled: "yes"
+  endpoint: "http://10.85.42.144"
```
Comment (shyamakirala) [inline runtime/src/main/resources/packaged/configuration.yml]: avoid committing IPs

Comment (amitkumar94) [reply runtime/src/main/resources/packaged/configuration.yml]: was needed for the new tests to pass, will remove now.

File: client/pom.xml (hunk 1 above)
Comment (amitkumar94) [reply client/pom.xml]: still not merging in master, will create new tag from master, with correct version details.

**Pull request merged**